### ISSUE TITLE
Crash with new architecture

### ISSUE THREAD

**Issue by meghvarse** [wxik/react-native-rich-editor#331]
Getting crash after enable new architecture with react-native 0.72.3.

Message from debugger: Terminated due to signal 6

**Comment by androxxe** [comment]
facing same problem

**Comment by muratcanoguzhan** [comment]
Same!

**Comment by rohit9625** [comment]
Yes, the crash occurred because of new architecture when I was using `react-native-pell-rich-editor@1.9.0`. However, it doesn't occru on latest version, i.e., **1.10.0**
